.github/workflows/test-terra-jupyter-aou.yml

```diff
@@ -84,7 +84,7 @@ jobs:
           --workdir=/tests \
           --entrypoint="" \
           terra-jupyter-aou:smoke-test \
-          /bin/bash -c 'for nb in {terra-jupyter-python/tests,terra-jupyter-aou/tests}/*ipynb ; do jupyter nbconvert --to html --ExecutePreprocessor.allow_errors=True --execute "${nb}" ; done'
+          /bin/bash -c 'for nb in {terra-jupyter-python/tests,terra-jupyter-aou/tests}/*ipynb ; do jupyter nbconvert --to html_embed --ExecutePreprocessor.allow_errors=True --execute "${nb}" ; done'
```

Comment:
This was failing with:
```
nbconvert.exporters.base.ExporterNameError: Unknown exporter "html", did you mean one of: html_ch, html_embed, html_toc, selectLanguage?
```

I changed all instances of `--to hrml` to `--to html_embed`